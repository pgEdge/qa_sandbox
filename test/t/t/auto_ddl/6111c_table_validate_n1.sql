SELECT pg_sleep(1);--to ensure all objects are replicated

-- 6111c - Validate tables on n1

SET ROLE appuser;

SET search_path TO s611, public;
-- Validate sub_tx_table0
-- Expected: table does not exist
\d sub_tx_table0
SELECT * FROM get_table_repset_info('sub_tx_table0');

-- Validate sub_tx_table2
-- Expected: table does not exist
\d sub_tx_table2
SELECT * FROM get_table_repset_info('sub_tx_table2');

-- Validate sub_tx_table3
-- Expected: table does not exist
\d sub_tx_table3
SELECT * FROM get_table_repset_info('sub_tx_table3');

-- Validate sub_tx_table5, sub_tx_table5a, sub_tx_table5c, sub_tx_table5b should not exist
-- Expected: tables do not exist
\d sub_tx_table5
SELECT * FROM get_table_repset_info('sub_tx_table5');
\d sub_tx_table5a
SELECT * FROM get_table_repset_info('sub_tx_table5a');
\d sub_tx_table5b
SELECT * FROM get_table_repset_info('sub_tx_table5b'); -- should not exist
\d sub_tx_table5c
SELECT * FROM get_table_repset_info('sub_tx_table5c');

-- Validate table_ctas1
-- Expected: table does not exist
\d table_ctas1
SELECT * FROM get_table_repset_info('table_ctas1');

-- Validate table_ctas2
-- Expected: table does not exist
\d table_ctas2
SELECT * FROM get_table_repset_info('table_ctas2');

-- Validate table_ctas3
-- Expected: table does not exist
\d table_ctas3
SELECT * FROM get_table_repset_info('table_ctas3');

-- Validate table_ctas4
-- Expected: table does not exist
\d table_ctas4
SELECT * FROM get_table_repset_info('table_ctas4');

-- Validate table_ctas5
-- Expected: table does not exist
\d table_ctas5
SELECT * FROM get_table_repset_info('table_ctas5');

-- Validate table_ctas6
-- Expected: table does not exist
\d table_ctas6
SELECT * FROM get_table_repset_info('table_ctas6');

-- Validate table_si1
-- Expected: table does not exist
\d table_si1
SELECT * FROM get_table_repset_info('table_si1');

-- Validate table_si2
-- Expected: table does not exist
\d table_si2
SELECT * FROM get_table_repset_info('table_si2');

-- Validate table_si3
-- Expected: table does not exist
\d table_si3
SELECT * FROM get_table_repset_info('table_si3');

-- Validate table_si4
-- Expected: table does not exist
\d table_si4
SELECT * FROM get_table_repset_info('table_si4');

-- Validate table_si5
-- Expected: table does not exist
\d table_si5
SELECT * FROM get_table_repset_info('table_si5');

-- Validate table_l1
-- Expected: table does not exist
\d table_l1
SELECT * FROM get_table_repset_info('table_l1');

-- Validate table_l2
-- Expected: table does not exist
\d table_l2
SELECT * FROM get_table_repset_info('table_l2');

-- Validate table_l3
-- Expected: table does not exist
\d table_l3
SELECT * FROM get_table_repset_info('table_l3');

-- Validate table_l4
-- Expected: table does not exist
\d table_l4
SELECT * FROM get_table_repset_info('table_l4');

-- Validate table_l5
-- Expected: table does not exist
\d table_l5
SELECT * FROM get_table_repset_info('table_l5');

RESET ROLE;
--dropping the schema
DROP SCHEMA s611 CASCADE;