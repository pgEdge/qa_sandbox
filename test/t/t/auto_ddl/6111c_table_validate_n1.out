SELECT pg_sleep(1);--to ensure all objects are replicated
 pg_sleep 
----------
 
(1 row)

-- 6111c - Validate tables on n1
SET ROLE appuser;
SET
SET search_path TO s611, public;
SET
-- Validate sub_tx_table0
-- Expected: table does not exist
\d sub_tx_table0
Did not find any relation named "sub_tx_table0".
SELECT * FROM get_table_repset_info('sub_tx_table0');
 nspname | relname | set_name 
---------+---------+----------
(0 rows)

-- Validate sub_tx_table2
-- Expected: table does not exist
\d sub_tx_table2
Did not find any relation named "sub_tx_table2".
SELECT * FROM get_table_repset_info('sub_tx_table2');
 nspname | relname | set_name 
---------+---------+----------
(0 rows)

-- Validate sub_tx_table3
-- Expected: table does not exist
\d sub_tx_table3
Did not find any relation named "sub_tx_table3".
SELECT * FROM get_table_repset_info('sub_tx_table3');
 nspname | relname | set_name 
---------+---------+----------
(0 rows)

-- Validate sub_tx_table5, sub_tx_table5a, sub_tx_table5c, sub_tx_table5b should not exist
-- Expected: tables do not exist
\d sub_tx_table5
Did not find any relation named "sub_tx_table5".
SELECT * FROM get_table_repset_info('sub_tx_table5');
 nspname | relname | set_name 
---------+---------+----------
(0 rows)

\d sub_tx_table5a
Did not find any relation named "sub_tx_table5a".
SELECT * FROM get_table_repset_info('sub_tx_table5a');
 nspname | relname | set_name 
---------+---------+----------
(0 rows)

\d sub_tx_table5b
Did not find any relation named "sub_tx_table5b".
SELECT * FROM get_table_repset_info('sub_tx_table5b'); -- should not exist
 nspname | relname | set_name 
---------+---------+----------
(0 rows)

\d sub_tx_table5c
Did not find any relation named "sub_tx_table5c".
SELECT * FROM get_table_repset_info('sub_tx_table5c');
 nspname | relname | set_name 
---------+---------+----------
(0 rows)

-- Validate table_ctas1
-- Expected: table does not exist
\d table_ctas1
Did not find any relation named "table_ctas1".
SELECT * FROM get_table_repset_info('table_ctas1');
 nspname | relname | set_name 
---------+---------+----------
(0 rows)

-- Validate table_ctas2
-- Expected: table does not exist
\d table_ctas2
Did not find any relation named "table_ctas2".
SELECT * FROM get_table_repset_info('table_ctas2');
 nspname | relname | set_name 
---------+---------+----------
(0 rows)

-- Validate table_ctas3
-- Expected: table does not exist
\d table_ctas3
Did not find any relation named "table_ctas3".
SELECT * FROM get_table_repset_info('table_ctas3');
 nspname | relname | set_name 
---------+---------+----------
(0 rows)

-- Validate table_ctas4
-- Expected: table does not exist
\d table_ctas4
Did not find any relation named "table_ctas4".
SELECT * FROM get_table_repset_info('table_ctas4');
 nspname | relname | set_name 
---------+---------+----------
(0 rows)

-- Validate table_ctas5
-- Expected: table does not exist
\d table_ctas5
Did not find any relation named "table_ctas5".
SELECT * FROM get_table_repset_info('table_ctas5');
 nspname | relname | set_name 
---------+---------+----------
(0 rows)

-- Validate table_ctas6
-- Expected: table does not exist
\d table_ctas6
Did not find any relation named "table_ctas6".
SELECT * FROM get_table_repset_info('table_ctas6');
 nspname | relname | set_name 
---------+---------+----------
(0 rows)

-- Validate table_si1
-- Expected: table does not exist
\d table_si1
Did not find any relation named "table_si1".
SELECT * FROM get_table_repset_info('table_si1');
 nspname | relname | set_name 
---------+---------+----------
(0 rows)

-- Validate table_si2
-- Expected: table does not exist
\d table_si2
Did not find any relation named "table_si2".
SELECT * FROM get_table_repset_info('table_si2');
 nspname | relname | set_name 
---------+---------+----------
(0 rows)

-- Validate table_si3
-- Expected: table does not exist
\d table_si3
Did not find any relation named "table_si3".
SELECT * FROM get_table_repset_info('table_si3');
 nspname | relname | set_name 
---------+---------+----------
(0 rows)

-- Validate table_si4
-- Expected: table does not exist
\d table_si4
Did not find any relation named "table_si4".
SELECT * FROM get_table_repset_info('table_si4');
 nspname | relname | set_name 
---------+---------+----------
(0 rows)

-- Validate table_si5
-- Expected: table does not exist
\d table_si5
Did not find any relation named "table_si5".
SELECT * FROM get_table_repset_info('table_si5');
 nspname | relname | set_name 
---------+---------+----------
(0 rows)

-- Validate table_l1
-- Expected: table does not exist
\d table_l1
Did not find any relation named "table_l1".
SELECT * FROM get_table_repset_info('table_l1');
 nspname | relname | set_name 
---------+---------+----------
(0 rows)

-- Validate table_l2
-- Expected: table does not exist
\d table_l2
Did not find any relation named "table_l2".
SELECT * FROM get_table_repset_info('table_l2');
 nspname | relname | set_name 
---------+---------+----------
(0 rows)

-- Validate table_l3
-- Expected: table does not exist
\d table_l3
Did not find any relation named "table_l3".
SELECT * FROM get_table_repset_info('table_l3');
 nspname | relname | set_name 
---------+---------+----------
(0 rows)

-- Validate table_l4
-- Expected: table does not exist
\d table_l4
Did not find any relation named "table_l4".
SELECT * FROM get_table_repset_info('table_l4');
 nspname | relname | set_name 
---------+---------+----------
(0 rows)

-- Validate table_l5
-- Expected: table does not exist
\d table_l5
Did not find any relation named "table_l5".
SELECT * FROM get_table_repset_info('table_l5');
 nspname | relname | set_name 
---------+---------+----------
(0 rows)

RESET ROLE;
RESET
--dropping the schema
DROP SCHEMA s611 CASCADE;
INFO:  DDL statement replicated.
DROP SCHEMA
